SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslator.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "EngineClasses/SpatialVirtualWorkerTranslator.h"
+#include "EngineClasses/SpatialNetDriver.h"
+#include "Interop/Connection/SpatialWorkerConnection.h"
+#include "SpatialConstants.h"
+#include "Utils/SchemaUtils.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialVirtualWorkerTranslator);
+
+namespace {
+	// The translation schema is a list of Mappings, where each entry has a virtual and physical worker ID. 
+	inline void GetMappingFromSchema(Schema_Object* Object, Schema_FieldId Id, VirtualToPhysicalWorkerMap& Mapping)
```

Comment:
Is that a GDK style approach? I've always used anonymous namespaces because it doesn't clutter the .h file with things that users of the class need to know about.

Reply:
In a small class I think private methods are fine.

Reply:
But this is purely opinion rather than an issue of following a rigid style guide.
I don't think we really have a hard and fast rule for this.In this case I thought it might make the code a bit simpler as we can remove a parameter and this isn't such fine detail that it looks weird in a header.

Reply:
I went ahead and moved them into private methods and renamed. I don't care that much either way.